src/sonic-bgpcfgd/bgpcfgd/managers_bgp.py

```diff
@@ -160,11 +163,18 @@ def add_peer(self, vrf, nbr, data):
         print_data = vrf, nbr, data
         bgp_asn = self.directory.get_slot("CONFIG_DB", swsscommon.CFG_DEVICE_METADATA_TABLE_NAME)["localhost"]["bgp_asn"]
         #
-        lo0_ipv4 = self.get_lo0_ipv4()
+        lo0_ipv4 = self.get_lo_ipv4("Loopback0|")
```

Comment:
how about ipv6 loopback?

Reply:
we are using ipv4 loopback only as originator-id (4-byte) in template.